MonoGame.Framework.Content.Pipeline/Graphics/GraphicsUtil.cs

```diff
@@ -263,5 +273,241 @@ static void Compress(Type targetType, TextureContent content, bool generateMipMa
                 content.ConvertBitmapType(targetType);
             }
         }
+
+        // Compress the greyscale font texture page using a specially-formulated DXT3 mode
+        static unsafe void CompressFontDXT3(TextureContent content, bool generateMipmaps)
```

Comment:
We should make this public.  We will need to call it from the `TextureProfile`s for specific platforms like XB1, Vita, WiiU, etc.